doc/collections/_pages/content/setup--windows/Readme.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Section on Windows Server Workstation Setup
+category: Documentation
+content_type: Install & Setup
+published: false
+---
+
+<!-- Begin GitHub-Flavored Markdown (GFM)
+See: https://docs.github.com/get-started/writing-on-github
+Spec: https://github.github.com/gfm
+-->
+
+# Windows Server Workstation Setup
+
+This section deals with preparing devices for (and installing) Windows
+Server as the host OS for modern software development workstations.
+
+Although using Windows may not be a requirement for contributions, many have
+made the conscious decision not to go with the more common locked-down option
+of Apple products. Oftentimes, the hardware associated with the machine has
+the most compatibility with drivers specifically made for Windows while
+Linux driver support may be limited or non-existent in some cases.
```

Comment:
```suggestion
Linux driver support may be limited or non-existent in some cases.

A Windows as a software development workstation using one of the
"Home" or "Professional" offerings has been a notoriously bad
decision due to unexpected updates forcing the reboot at
inconvenient times and which is something that the Server
offerings are more prepared to avoid having this happen.
```